db/columns/operations/create.py

```diff
@@ -66,6 +68,15 @@ def create_column(engine, table_oid, column_data):
         engine
     )
 
+def gen_col_name(table):
+    col_num = None
+    base_name = COLOMN_NAME_TEMPLATE
+    col_num = len(table.sa_column_names)
+    name = f'{base_name} {col_num}'
+    if len(name) > POSTGRES_COL_NAME_LEN_CAP:
```

Comment:
I don't think you're going to need to worry about this check. `Column <number>` is not going to be longer than the length limit. I assume you copied this from the table name generation - it is needed there because we get the base name of the table from the file that's used to create it and the file name can be arbitrarily long. Here, the base name is just `Column`.